src/silverlabnwb/metadata_gui.py

```diff
@@ -394,40 +396,55 @@ def record_expt(self):
                 elif field in expt:
                     del expt[field]
 
-    def make_label(self, parent, name, row=0, sticky='w'):
+    def make_label(self, parent, name, row=0, sticky='w', tooltip_text=''):
         """Make the human-friendly label for a form section."""
         name = name.capitalize().replace('_', ' ') + ':'
-        ttk.Label(parent, text=name).grid(row=row, column=0, sticky=sticky)
+        label = ttk.Label(parent, text=name)
+        label.grid(row=row, column=0, sticky=sticky)
+        if len(tooltip_text) > 0:
+            Hovertip(label, tooltip_text, hover_delay=1000)  # delay in ms
 
     def make_expts_part(self, parent, part_name):
         """Make a component frame for the experiment editor."""
         frame = ttk.Frame(parent)
-        self.make_label(frame, part_name)
         if isinstance(self.template_expt[part_name], collections.Mapping):
             # This is actually a related group of fields
+            self.make_label(frame, part_name)
             self.expts_boxes[part_name] = boxes = {}
-            self.make_expts_fields(frame, self.template_expt[part_name], boxes)
+            self.make_expts_fields(frame, self.template_expt[part_name], boxes, part_name)
         else:
+            self.make_label(frame, part_name, tooltip_text=self.template_expt_comments[part_name])
             textbox = T.Text(frame, width=100, height=5, wrap='word')
             textbox.grid(row=1, column=0, sticky='nesw')
-            textbox.insert('1.0', self.template_expt[part_name])
+            expts_part = self.template_expt[part_name]
+            if expts_part is not None:
+                textbox.insert('1.0', expts_part)
+            else:
+                textbox.insert('1.0', "")
```

Comment:
This could be a bit shorter but perhaps less clear:
```suggestion
            textbox.insert('1.0', expts_part if expts_part is not None else ""):
 ```